index.ejs

```diff
@@ -4,9 +4,17 @@
     <meta charset="utf-8">
     <title>cp-zen-frontend</title>
     <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, user-scalable=no">
+    <% if (htmlWebpackPlugin.options.openGraph) { %>
+      {{#each openGraphProperties}}
+        {{#each this}}
+          <meta name="og:{{@../key}}" content="{{this}}">
+        {{else}}
+          <meta name="og:{{@key}}" content="{{this}}">
+        {{/each}}
+      {{/each}}
+    <% } %>
     <link rel="stylesheet" href="//fonts.googleapis.com/css?family=Lato:300,400,700" type="text/css" />
     <script src="https://www.google.com/recaptcha/api.js?onload=vueRecaptchaApiLoaded&render=explicit" async defer></script>
-    <link rel="stylesheet" href="./assets/style.css" type="text/css" />
```

Comment:
The file doesn't exist, and has been resulting in a 404 since the start of the project 😅  It also caused issues with OpenGraph as it was a relative URL, and so Hapi picked it up as a Dojo URL